sdk/digitaltwins/azure-digitaltwins-core/README.md

```diff
@@ -65,7 +75,7 @@ You can learn more about Azure Digital Twins by visiting [Azure Digital Twins Do
 
 ## Examples
 
-You can explore the digital twins APIs (using the client library) using the samples project.
+You can explore the digital twins APIs (using the client library) using the samples project. More samples on DigitalTwins are available [here](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/digitaltwins/azure-mgmt-digitaltwins/generated_samples).
```

Comment:
I'm surprised the generated samples aren't part of the samples project already. Can we not add them instead? Having samples in two places is a bit confusing.

Reply:
The currently existing docs, helps in getting started with key elements of Azure Digital Twins, such as creating Azure Digital Twins instances, models, twin graphs, etc. Also the samples provided here guides to familiarize yourself with the various APIs that help you program against Azure Digital Twins. So we dont want to include all of the generated 30+ samples from the github here.

Reply:
It may be confusing to involve samples of mgmt SDK in data-plane SDK readme.md

Reply:
@msyyc That makes sense. So I have reverted it.